bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -610,7 +610,7 @@ public int getExplictLacInterval() {
     /**
      * Set the interval to check the need for sending an explicit LAC.
      * @param interval
-     *        Number of seconds between checking the need for sending an explict LAC.
+     *        Number of milli seconds between checking the need for sending an explict LAC.
      * @return Client configuration.
      */
     public ClientConfiguration setExplictLacInterval(int interval) {
```

Comment:
@revans2 thanks for the review. I have addressed the local variable naming issue.

This feature has been contributed by SalesForce by merging their local fork. @jvrao are you ok with this change ?

The feature on the "Apache version" is very "new" and so I think there is no compatibility issue actually.

I have filed the PR now because I would like to address the issue as soon as possible, so that we are sure that no one is using it in production